lib/pgut/pgut-spi.c

```diff
@@ -84,6 +84,8 @@ execute_with_args(int expected, const char *src, int nargs, Oid argtypes[], Datu
 	int		i;
 	char	c_nulls[FUNC_MAX_ARGS];
 
+    	memset(c_nulls, 0, sizeof(c_nulls));
```

Comment:
W/o this change tests are failing with `error: ‘c_nulls’ may be used uninitialized [-Werror=maybe-uninitialized]`